requirements.txt

```diff
@@ -1,11 +1,7 @@
 praw==4.4.0
 redis<3.0.0
-hiredis
-better_exceptions
```

Comment:
Even if we don't use it in production, having this available for debugging is incredibly useful. Is there a reason we want to nuke it?